bin/tic_tac_toe.rb

```diff
@@ -0,0 +1,15 @@
+require_relative './../lib/engine.rb'
+require_relative '../bin/user_interface.rb'
+# game running class
+class Game
+  attr_accessor :board
+  game = TicTacToe.new
```

Comment:
It's not advisable to directly call another class, let alone initialize it, inside another. This creates too tight a coupling (dependence) of your Game class to the TicTacToe and Engine classes. What if there are changes into TicTacToe and Engine? For instance what if you decided later on that TicTacToe should be initialized with parameters? Then you would need to also change Game class. So I suggest you simply pass instantiated TicTacToe and Engine objects as parameters to Game class. Which means you need to have an initialize method for Game class.